Code/Framework/AzFramework/Platform/Linux/AzFramework/API/ApplicationAPI_Linux.h

```diff
@@ -25,4 +29,22 @@ namespace AzFramework
 
         using Bus = AZ::EBus<LinuxLifecycleEvents>;
     };
+
+#if AZ_LINUX_WINDOW_MANAGER_XCB
+    class LinuxXCBConnectionManager
+       : public AZ::EBusTraits
+    {
+    public:
+        // Bus Configuration
+        static const AZ::EBusHandlerPolicy HandlerPolicy = AZ::EBusHandlerPolicy::Single;
+        static const AZ::EBusAddressPolicy AddressPolicy = AZ::EBusAddressPolicy::Single;
+
+        virtual ~LinuxXCBConnectionManager() {}
+
+        using Bus = AZ::EBus<LinuxXCBConnectionManager>;
+
+        virtual xcb_connection_t* GetXCBConnection() const = 0;
+
+    };
+#endif // AZ_LINUX_WINDOW_MANAGER_XCB
```

Comment:
Exposing an AZ Interface instead of an EBus would provide a more natural API for users
```suggestion
#if AZ_LINUX_WINDOW_MANAGER_XCB
    class LinuxXCBConnectionManager
    {
    public:
        virtual ~LinuxXCBConnectionManager() = default;

        virtual xcb_connection_t* GetXCBConnection() const = 0;
    };
    
     class LinuxXCBConnectionManagerBusTraits
        : public AZ::EBusTraits
    {
    public:
        //////////////////////////////////////////////////////////////////////////
        // EBusTraits overrides
        static constexpr AZ::EBusHandlerPolicy HandlerPolicy = AZ::EBusHandlerPolicy::Single;
        static constexpr AZ::EBusAddressPolicy AddressPolicy = AZ::EBusAddressPolicy::Single;
        //////////////////////////////////////////////////////////////////////////
    };
    
    using LinuxXCBConnectionManagerBus = AZ::EBus<LinuxXCBConnectionManager, LinuxXCBConnectionManagerBusTraits>;
    using LinuxXCBConnectionManagerInterface = AZ::Interface<LinuxXCBConnectionManager>;
#endif // AZ_LINUX_WINDOW_MANAGER_XCB
```

Reply:
Very nice, will update this